content/en/docs/Reference/configuration.md

```diff
@@ -196,7 +198,8 @@ upgrade:
   poweroff: true
 
   # Use a different source for the upgrade
-  source: "oci:.."
+  system:
```

Comment:
I think this should be `recovery-system`: https://github.com/kairos-io/kairos-agent/blob/a066e322264ea42e29a3edf5c55eec2e03b4c0ec/pkg/implementations/spec/spec.go#L45-L46